clients/clients.yaml

```diff
@@ -0,0 +1,16 @@
+ndoli:
+  tests:
+    git-annex-home: |
+      set -ex
+      mkdir -p "$HOME/git-annex-ci/$BUILDNO"
+      cd "$HOME/git-annex-ci/$BUILDNO"
+      git annex version
+      timeout 1800 git annex test
```

Comment:
something should `rm` those `$BUILDNO` directories here and below

Reply:
Done.